lib/author.rb

```diff
@@ -0,0 +1,39 @@
+class Author
+  attr_reader :first_name,
+              :last_name,
+              :books
+
+  def initialize(attributes)
+    @first_name = attributes[:first_name]
+    @last_name = attributes[:last_name]
+    @books = []
+  end
+
+  def name
+    "#{@first_name} #{@last_name}"
+  end
+
+  def add_book(book)
+    @books << book
+  end
+
+  def class
+    @book
+  end
+
+  def title
+    @book.title
```

Comment:
Here, `@book` is going to return `nil`.  Similar to `class` above, we don't need to recreate these methods - `title` is a book method - it does not need to be (and should not be) recreated on an author.